ddpui/tests/core/test_llm_summarization.py

```
@@ -0,0 +1,265 @@
+
+            # Call without orguser_id
+            try:
+                summarize_logs(
+                    orguser_id=None,
+                    type=LogsSummarizationType.DEPLOYMENT,
+                    flow_run_id="test-flow-run",
+                    task_id="task-1",
+                )
+            except Exception:
+                # We expect it to fail at some point, but we're testing the org extraction
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)